### PR TITLE
added more correct story carousel

### PR DESCRIPTION
**Pull request by HC-Hwang**
Added a carousel in the Story section, using ddmail2009's new JS carousel function.

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -392,6 +392,7 @@ class Carousel {
 }
 
 var myCarousel = [new Carousel($("#CarouselPhoto"))]
```
Comment (ddmail2009) [inline assets/js/main.js]: Just append to the list.

File: assets/js/main.js [hunk 2]
```diff
@@ -421,14 +422,7 @@ var initialize = function() {
     $(window).on("resize", () => window.requestAnimationFrame(animateUpdate))
     $(document).on("scroll", () => window.requestAnimationFrame(animateUpdate))
 
-    // manual gif
```
Comment (ddmail2009) [inline assets/js/main.js:424]: nice.

File: assets/js/main.js (hunk 1 above)
Comment (ddmail2009) [reply assets/js/main.js]: If you want to use two list, then you have to modify line249 too.

Comment (HC-Hwang) [reply assets/js/main.js]: Now appended to the list.

File: assets/css/style.css [hunk 3]
```diff
@@ -340,5 +340,5 @@ a {
 
 .nav-link {
     --bs-nav-link-hover-color: rgb(120 166 197);
-    --bs-nav-link-color: rgb(51 135 192);
+    --bs-nav-link-color: rgb(60 101 129);
```
Comment (ddmail2009) [inline assets/css/style.css:343]: this is unrelated

File: assets/js/main.js [hunk 4]
```diff
@@ -391,8 +391,7 @@ class Carousel {
     }
 }
 
-var myCarousel = [new Carousel($("#CarouselPhoto"))]
-var myCarouselStory = [new Carousel($("#CarouselStory"))]
+var myCarousel = [new Carousel($("#CarouselPhoto")), new Carousel($("#CarouselStory"))]
```
Comment (ddmail2009) [inline assets/js/main.js:394]: good
